shell/platform/windows/cursor_handler.cc

```diff
@@ -11,9 +11,22 @@
 static constexpr char kChannelName[] = "flutter/mousecursor";
 
 static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
-
 static constexpr char kKindKey[] = "kind";
 
+// [kSetCustomCursorMethod] allows developers to set a custom cursor with
+// rawRGBA buffer.
+static constexpr char kSetCustomCursorMethod[] = "setCustomCursor";
```

Comment:
Let's change it to "setCustomCursor/windows" to avoid confliction in the future.